src/index.js

```diff
@@ -32,8 +33,9 @@ export default ({ types: t }) => ({
         const svgPath = join(dirname(iconPath), path.node.source.value);
         const svgSource = readFileSync(svgPath, 'utf8');
         const optimizedSvgSource = optimize(svgSource);
+        const escapeSvgSource = escape(optimizedSvgSource);
```

Comment:
Can we name this something to avoid confusion with the deprecated escape global? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/escape